classes/classes/Scenes/Areas/GlacialRift/AmarokScene.as

```diff
@@ -0,0 +1,77 @@
+			outputText("\n\nYou spot your " + player.armorName + " a bit away. It must've been ripped off you when you were passed out. You scramble to it and put it back on, then head back to camp as fast as you can.", false);
+			doNext(camp.returnToCampUseOneHour);
+		}
+
+		public function amarokBadEnd():void { //Hungry Hungry Hipp-- err, Wolves
+			clearOutput();
+			outputText("You've been here before. The Amarok will carry you to its den, let its pups deal with you as they will, and it'll vanish off somewhere. You'll wake up later and flee before it returns or its pups get hungry. You close your eyes and wait for it to pick you up. You just want to get this over with.", false);
+			outputText("\n\nThe Amarok, however, has other plans. Instead of coming behind you and picking you up, it hits you with a paw that rolls you onto your back, then puts said paw on your chest. You wheeze under its weight. It managed to step in just the right spot to knock your breath out of you. As though the air wasn't freezing enough, a wave of cold horror washes through your spine. You don't know what's going to happen, but you don't like it.", false);
+			outputText("\n\nYou writhe under its grasp as your throat closes in terror. Your efforts are fruitless, and the Amarok watches you in mild amusement. Your blood starts to pound in your ears. You need something-- anything --that'll help you get away. You flop an arm at the leg holding you down, praying it'll knock the Amarok off. It doesn't work. You try to get a foot under it and kick it off. You can't even roll your legs up. You try to scream for help in a futile hope anyone is nearby. You instead choke on your own breath. The Amarok's cruel gaze watches you with glee.", false);
+			outputText("\n\nYou stop thinking. You just want air.", false);
+			outputText("\n\nLines of pain well in your abdomen and you take in the biggest breath of your life-- only to promptly release it with a screech. You instinctively curl up to try and protect yourself, but the Amarok takes a hold of one of your arms and shakes you violently. It only releases you when it accidentally rips off part of your " + player.armorName + ". You get flung a bit away and take a small tumble. Before you can get up, though, the Amarok is on you. It holds you down with one of its paws and begins to tear off the rest of your " + player.armorName + ", carelessly tossing it aside. You jerk, and roll, and rip at its leg, but it refuses to release you. Instead, it opens its jaws and snaps them shut around your neck.", false);
+			getGame().gameOver();
+		}
+		
+		public function winAgainstAmarok():void {
+			flags[kFLAGS.AMAROK_LOSSES] = 0; //Kinda unfair to continually stack it imo. I'd say subtract 1 but idk how to do that
```

Comment:
Ah, yeah, when I first wrote this I didn't know about that, forgot to update it now that I learned. Thank you though xD